resources/views/profile_complete.blade.php

```diff
@@ -0,0 +1,116 @@
+@extends('app') @section('content')
+<div class="">
+  <div class="container">
+    <div class="col-lg-12">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-6 col-md-3 col-md-offset-3">
+            <div class="panel panel-default">
+              <div class="panel-heading">
+                <strong>Complete Your Profile</strong>
+              </div>
+              <div class="panel-body">
+                <form class="form-horizontal" role="form" method="POST"
+                  action="{{ url('profile_complete') }}"
+                  enctype="multipart/form-data" id="upload">
+                  <input type="hidden" name="_token"
+                    value="{{ csrf_token() }}">
+                  <fieldset>
+                    <div class="row">
+                      @if (count($errors) > 0)
```

Comment:
This forms are getting unnecessarily long. Abstract the showing of errors into a `partial` blade file. and just `@include` here